dbschema/1.tables.sql

```diff
@@ -11,7 +11,11 @@ CREATE TABLE tbl_token_transfers (
     gas_limit NUMERIC(256, 0) NOT NULL,
     gas_price NUMERIC(256, 0) NOT NULL,
     block_number INTEGER NOT NULL,
-    token_type TEXT NOT NULL
+    token_type TEXT NOT NULL,
+    is_multi_address BOOLEAN NOT NULL,
```

Comment:
@Femi-Ogunkola these should be NULLABLE for now - create a separate PR where you make them not nullable (here and in the code)